src/pages/index.js

```diff
@@ -71,10 +71,7 @@ function Home() {
               </Link>
             </div>
           </div>
-          <picture className={styles.heroImage}>
-            <source media="(min-width:662px)" srcSet="images/strykerman.png"></source>
-            <img src="images/gun.png" alt="Strykerman"></img>
-          </picture>
+          <img src="images/StrykerMan.svg" alt="Strykerman" className="stryker-man"></img>
```

Comment:
You can probably still use [picture](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) for svg's as well if the size is set properly

Reply:
Well it did not show the image... Only by removing the picture tag I got it to show. I'll try something else with the picture tag